sdk/sdk.go

```diff
@@ -63,8 +59,8 @@ func createBlockVotesQueryData(queryParams *L2Block) ([]byte, error) {
 	return data, nil
 }
 
-func (babylonClient *BabylonQueryClient) queryListOfVotedFinalityProviders(queryParams *L2Block) ([]string, error) {
-	queryData, err := createBlockVotesQueryData(queryParams)
+func (babylonClient *BabylonQueryClient) queryListOfVotedFinalityProviders(queryParams *L2Block) (*[]string, error) {
```

Comment:
```suggestion
func (babylonClient *BabylonQueryClient) queryListOfVotedFinalityProviders(queryParams *L2Block) ([]string, error) {
```

iirc, in Go, it is unnecessary to return the pointer of a slice as it is already a pointer

Reply:
i think you r right. slice is a pointer

Reply:
cc @lesterli